pkg/codegen/docs/examples.go

```diff
@@ -46,6 +46,9 @@ func (dctx *docGenContext) decomposeDocstring(docstring string) docInfo {
 	if docstring == "" {
 		return docInfo{}
 	}
+	if strings.Contains(docstring, "<!--Begin TFConversion -->") {
```

Comment:
Let's make sure this is an exported constant, and not hard code strings here.

```suggestion
	if strings.Contains(docstring, BeginCodeChooserBlock) {
```